packages/libs/react-components/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,59 @@
+import { SystemIcons } from '../Icons';
+
+import {
+  StyledIconWrapper,
+  StyledInput,
+  StyledInputWrapper,
+  StyledLeadingText,
+} from './styles';
+
+import classnames from 'classnames';
+import React, { FC } from 'react';
+
+export interface IInputProps
+  extends Omit<
+    React.HTMLAttributes<HTMLInputElement>,
+    'as' | 'disabled' | 'children'
```

Comment:
I think we want to omit the classname here as well. we want to be strict in styling

Reply:
by omitting classnames we won't be able to extend a component in styles, cause all styles are applied by classname under the hood of stitches, therefore if we omit it, and then extend it as a styled component, we won't be able to overwrite any styles